changelog/1.6.0_2021-02-16/mentix-auth-fix.md

```diff
@@ -0,0 +1,5 @@
+Bugfix: Site authorization status changes
```

Comment:
This file should not be changed, only the unreleased one.

Reply:
That file actually went there by accident (and I wondered why the initial changelog had disappeared...). Removed the file (it never was released, it was just placed in the wrong directory).